src/shared/components/CSVExportButton.tsx

```diff
@@ -52,8 +52,10 @@ class CSVExportButton extends PureComponent<StateProps, {}> {
 
   private handleClick = () => {
     const {files} = this.props
+    const formatter = createDateTimeFormatter('YYYY-MM-DD HH:mm')
     const csv = files.join('\n\n')
-    const now = moment().format('YYYY-MM-DD-HH-mm')
+    let now = formatter.format(new Date())
```

Comment:
it's probably slightly better to declare `now` as a `const` and replace the spaces when we initialize it:
```ts
formatter.format(new Date()).replace(/\s+/gi, '_')
```
but this is fine for now